requirements.txt

```diff
@@ -2,4 +2,5 @@ Mako
 PyYAML
 six
 pbr
-defusedxml
+# For OpenNebula provider
+# defusedxml
```

Comment:
In fact looking at https://docs.openstack.org/pbr/latest/user/using.html#setup-py it seems that extra requirements should go to setup.cfg instead of the requirements.txt, it seems I have mixed some stuff during my tests. I don't know if we want to use setup.cfg in addition of requirements.txt to list the dependencies